<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8">

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>Text Labeller</title>
</head><body>
<script type="text/javascript" src="foo_files/rangy-core.js"></script>
<script language="javascript"><!--
/* Script removed by snapshot save */
--></script>
<script type="text/javascript" src="foo_files/floating.js"></script>
<script type="text/javascript"><!--
/* Script removed by snapshot save */
--></script>
<table border="1">
<tbody><tr>
<td><div tagged="yes" id="tag0">note</div></td>
<td><div id="line0"><span style="background-color: yellow;">LETTERS |XML| xmlLoc_0 xmlBold_no xmlItalic_no xmlFontSize_largest-1 xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</span></div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag1">title</div></td>
<td><div id="line1"><span style="background-color: yellow;">5′-end SAGE for the analysis of transcriptional start sites |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_largest0 xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</span></div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag2">author</div></td>
<td><div id="line2">Shin-ichi Hashimoto1, Yutaka Suzuki2,4, Yasuhiro Kasai3,4, Kei Morohoshi1, Tomoyuki Yamada3, Jun Sese3, |XML| xmlLoc_1 xmlBold_yes xmlItalic_no xmlFontSize_largest-2 xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag3">author</div></td>
<td><div id="line3"><span style="background-color: yellow;">Shinichi Morishita3, Sumio Sugano2 &amp; Kouji Matsushima1 |XML| xmlLoc_1 xmlBold_yes xmlItalic_no xmlFontSize_largest-2 xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</span></div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag4">bodyText</div></td>
<td><div id="line4">Identification of the mRNA start site is essential in establish- |XML| xmlLoc_2 xmlBold_yes xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag5">bodyText</div></td>
<td><div id="line5">ing the full-length cDNA sequence of a gene and analyzing its |XML| xmlLoc_2 xmlBold_yes xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag6">bodyText</div></td>
<td><div id="line6">promoter region, which regulates gene expression. Here we |XML| xmlLoc_2 xmlBold_yes xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag7">bodyText</div></td>
<td><div id="line7">describe the development of a 5′-end serial analysis of gene |XML| xmlLoc_2 xmlBold_yes xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag8">bodyText</div></td>
<td><div id="line8">expression (5′ SAGE) that can be used to globally identify |XML| xmlLoc_2 xmlBold_yes xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag9">bodyText</div></td>
<td><div id="line9">transcriptional start sites and the frequency of individual |XML| xmlLoc_2 xmlBold_yes xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag10">bodyText</div></td>
<td><div id="line10">mRNAs. Of the 25,684 5′ SAGE tags in the HEK293 human |XML| xmlLoc_2 xmlBold_yes xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag11">bodyText</div></td>
<td><div id="line11">cell library, 19,893 matched to the human genome. Among |XML| xmlLoc_3 xmlBold_yes xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag12">bodyText</div></td>
<td><div id="line12">15,448 tags in one locus of the genome, 85.8%–96.1% of the |XML| xmlLoc_3 xmlBold_yes xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag13">bodyText</div></td>
<td><div id="line13">5′ SAGE tags were assigned within –500 to +200 nt of mRNA |XML| xmlLoc_3 xmlBold_yes xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag14">bodyText</div></td>
<td><div id="line14">start sites using the RefSeq, UniGene and DBTSS databases. |XML| xmlLoc_3 xmlBold_yes xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag15">bodyText</div></td>
<td><div id="line15">This technique should facilitate 5′-end transcriptome analysis |XML| xmlLoc_3 xmlBold_yes xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag16">bodyText</div></td>
<td><div id="line16">in a variety of cells and tissues. |XML| xmlLoc_3 xmlBold_yes xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag17">bodyText</div></td>
<td><div id="line17">Comprehensive analysis of gene expression in different cell sub- |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag18">bodyText</div></td>
<td><div id="line18">populations and microanatomic structures provides insight into |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag19">bodyText</div></td>
<td><div id="line19">human development and physiology. Recently developed functional |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag20">bodyText</div></td>
<td><div id="line20">genomic technologies such as cDNA microarrays1 and serial analysis of |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag21">bodyText</div></td>
<td><div id="line21">gene expression (SAGE)2 allow for the analysis of the expression of |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag22">bodyText</div></td>
<td><div id="line22">thousands of genes. The SAGE method has proved to be a useful tool |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag23">bodyText</div></td>
<td><div id="line23">for quantifying, cataloging and comparing the expression of genes in |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag24">bodyText</div></td>
<td><div id="line24">cells and tissues that are in various physiological, developmental and |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag25">bodyText</div></td>
<td><div id="line25">pathological states3–5. |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag26">bodyText</div></td>
<td><div id="line26">In the human genome project6, genes were predicted based on infor- |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag27">bodyText</div></td>
<td><div id="line27">mation obtained from expressed sequence tag (EST) maps, analysis of |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag28">bodyText</div></td>
<td><div id="line28">full-length cDNAs and computational annotation by Genscan, Genie, |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag29">bodyText</div></td>
<td><div id="line29">FGENES and other programs. However, these computational analyses |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag30">bodyText</div></td>
<td><div id="line30">had limitations. For example, they were unable to provide definitive |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag31">bodyText</div></td>
<td><div id="line31">evidence that a hypothetical gene was actually expressed. The Long |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag32">bodyText</div></td>
<td><div id="line32">SAGE method is useful in rapidly identifying not only novel genes and |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag33">bodyText</div></td>
<td><div id="line33">exons but also the frequency of each transcript. However, a remaining |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag34">bodyText</div></td>
<td><div id="line34">challenge in human genome expression profile analysis is the exact iden- |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag35">bodyText</div></td>
<td><div id="line35">tification and annotation of entire expressed genes based on informa- |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag36">bodyText</div></td>
<td><div id="line36">tion concerning their 5′ -end to 3′ -end transcripts. Therefore, a method |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag37">bodyText</div></td>
<td><div id="line37">for precisely and globally identifying the 5′ ends of transcripts is needed. |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag38">bodyText</div></td>
<td><div id="line38">Such a 5′-end capping method7 would give us the sequence tag to the |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag39">bodyText</div></td>
<td><div id="line39">first base of an mRNA. Using cDNA libraries made from oligo-capped |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag40">bodyText</div></td>
<td><div id="line40">mRNA preparations and SAGE, we developed a method to comprehen- |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag41">bodyText</div></td>
<td><div id="line41">sively identify the transcriptional start sites of mRNAs. |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag42">bodyText</div></td>
<td><div id="line42">The method is shown in Figure 1. The mRNA is split into two ali- |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag43">bodyText</div></td>
<td><div id="line43">quots and its cap structure is enzymatically replaced with two types of |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag44">bodyText</div></td>
<td><div id="line44">synthetic oligonucleotides containing MmeI, a type-IIS restriction |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag45">bodyText</div></td>
<td><div id="line45">endonuclease site, and the XhoI restriction enzyme site. Oligo- |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag46">bodyText</div></td>
<td><div id="line46">capping mRNA is then converted into first strand cDNA with a random |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag47">bodyText</div></td>
<td><div id="line47">adapter-primer. The second strand is synthesized using biotin-bound |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag48">bodyText</div></td>
<td><div id="line48">5′-primer and random adaptor-primer by PCR. The double-stranded |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag49">bodyText</div></td>
<td><div id="line49">cDNA is then cleaved with MmeI, which cleaves 20 bp away from its |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag50">bodyText</div></td>
<td><div id="line50">recognition site. After the 5′ -cDNAs are isolated by binding to strepta- |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag51">bodyText</div></td>
<td><div id="line51">vidin beads, the two pools of tags are ligated to each other. The subse- |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag52">bodyText</div></td>
<td><div id="line52">quent procedure for concatemerization is carried out according to the |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag53">bodyText</div></td>
<td><div id="line53">original SAGE protocol2. |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag54">bodyText</div></td>
<td><div id="line54">Using this method, we characterized 25,684 transcripts expressed |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag55">bodyText</div></td>
<td><div id="line55">in HEK293 cells as a test cell line and compared them to the human |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag56">bodyText</div></td>
<td><div id="line56">genome sequence. A total of 19,893 tags matched perfectly to genomic |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag57">bodyText</div></td>
<td><div id="line57">sequences representing 13,404 different tags (Table 1). Eighty percent |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag58">bodyText</div></td>
<td><div id="line58">(10,706 tags) of 13,404 different tags were assigned to unique posi- |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag59">bodyText</div></td>
<td><div id="line59">tions. Eleven percent (1,483 tags) of the tags matched two loci in the |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag60">bodyText</div></td>
<td><div id="line60">genome, 8.1% (1,090 tags) matched 3–99 loci, and 0.9% (125 tags) |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag61">bodyText</div></td>
<td><div id="line61">matched &gt;100 loci. The tags that mapped to multiple genomic loci |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag62">bodyText</div></td>
<td><div id="line62">mostly corresponded to retrotransposon elements, repetitive sequences, |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag63">bodyText</div></td>
<td><div id="line63">or pseudogenes. |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag64">bodyText</div></td>
<td><div id="line64">We estimated whether the 5′ SAGE tags matched to the mRNA |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag65">bodyText</div></td>
<td><div id="line65">start sites using three databases: the reference sequence database |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag66">bodyText</div></td>
<td><div id="line66">(RefSeq), the Gene Resource Locator (GRL) database, which assembles |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag67">bodyText</div></td>
<td><div id="line67">gene maps that include information on cis-elements in regulatory |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag68">bodyText</div></td>
<td><div id="line68">regions and alternatively spliced transcripts, and the database of |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag69">bodyText</div></td>
<td><div id="line69">Transcriptional Start Sites (DBTSS)8, which contains systematic 5′-end |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag70">bodyText</div></td>
<td><div id="line70">sequences of human full-length cDNAs. The distance distributions and |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag71">bodyText</div></td>
<td><div id="line71">the number and ratio of small-distance tag occurrences that we |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag72">bodyText</div></td>
<td><div id="line72">obtained are shown in Figure 2a and Table 2. The data indicate that our |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag73">bodyText</div></td>
<td><div id="line73">5′ SAGE tags coincided well with start site information for each data- |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag74">bodyText</div></td>
<td><div id="line74">base. Most (85.8%–98.2%) tags that mapped to each database were |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag75">bodyText</div></td>
<td><div id="line75">–500 to +200 nt of mRNA start sites. Notably, 23.5%–49.3% of 5′ SAGE |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag76">bodyText</div></td>
<td><div id="line76">tags hit the upstream regions of the defined transcription start sites |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag77">bodyText</div></td>
<td><div id="line77">(TSS) in these databases. |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag78">bodyText</div></td>
<td><div id="line78">To experimentally confirm the 5′ ends of known genes identified by |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag79">bodyText</div></td>
<td><div id="line79">5′ SAGE, we compared the directed sequencing data of the 5′ ends of |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag80">bodyText</div></td>
<td><div id="line80">captured full-length cDNAs in HEK293 with 5′ SAGE data. The repre- |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag81">bodyText</div></td>
<td><div id="line81">sentative TSS identified by 5′ SAGE and full-length cDNA using the |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag82">bodyText</div></td>
<td><div id="line82">oligo-capping method are shown in Supplementary Figure 1. A part of |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag83">bodyText</div></td>
<td><div id="line83">various 5′ SAGE tags corresponded to those of full-length cDNA using |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag84">bodyText</div></td>
<td><div id="line84"><span style="background-color: yellow;">the oligo-capping method. |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</span></div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag85">affiliation</div></td>
<td><div id="line85">1Department of Molecular Preventive Medicine, School of Medicine, 2Department of Virology, Institute of Medical Science,3Department of Computational Biology, |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag86">affiliation</div></td>
<td><div id="line86"><span style="background-color: yellow;">Graduate School of Frontier Science, The University of Tokyo, 7-3-1, Hongo, Bunkyo-ku, Tokyo 113-0033, Japan. 4These authors contributed equally to this work. |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</span></div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag87">note</div></td>
<td><div id="line87">Correspondence should be addressed to K.M. (koujim@m.u-tokyo.ac.jp). |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag88">note</div></td>
<td><div id="line88"><span style="background-color: yellow;">Published online 8 August 2004; doi:10.1038/nbt998 |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</span></div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag89">footnote</div></td>
<td><div id="line89"><span style="background-color: yellow;">1146	VOLUME 22 NUMBER 9 SEPTEMBER 2004 NATURE BIOTECHNOLOGY |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_yes xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</span></div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag90">note</div></td>
<td><div id="line90"><span style="background-color: yellow;">LETTERS |XML| xmlLoc_0 xmlBold_no xmlItalic_no xmlFontSize_largest-1 xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</span></div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag91">figureCaption</div></td>
<td><div id="line91">Figure 1 Scheme for construction of a 5′-end SAGE library as detailed |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag92">figureCaption</div></td>
<td><div id="line92"><span style="background-color: yellow;">in Methods. |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</span></div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag93">bodyText</div></td>
<td><div id="line93">We examined the nucleotide preference at the TSS. It has been |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag94">bodyText</div></td>
<td><div id="line94">reported9 that the nucleotides of TSS are A (47%), G (28%), C (14%) |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag95">bodyText</div></td>
<td><div id="line95">and T (12%) using 5,880 mRNAs in 276 human genes. Our data showed |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag96">bodyText</div></td>
<td><div id="line96">very similar percentages for the first nucleotide: A (41%), G (32%), |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag97">bodyText</div></td>
<td><div id="line97">C (17%) and T (10%). Taken together, these results suggest that our |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag98">bodyText</div></td>
<td><div id="line98">5′ SAGE method can precisely identify TSS. Thus, the method allows |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag99">bodyText</div></td>
<td><div id="line99">us to obtain both accurate transcriptional start site information and |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag100">bodyText</div></td>
<td><div id="line100">information that could be used to analyze the relationship between pro- |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag101">bodyText</div></td>
<td><div id="line101">moter and transcription. |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag102">bodyText</div></td>
<td><div id="line102">Interestingly, no matches to genome tags in the databases were found |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag103">bodyText</div></td>
<td><div id="line103">for 23% of the 5′ SAGE tags sequenced in this study. In 39% of these, the |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag104">bodyText</div></td>
<td><div id="line104">first nucleotide was A. Some of the tags without matches to the genome |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag105">bodyText</div></td>
<td><div id="line105">may contain single nucleotide mutations or deletions. |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag106">bodyText</div></td>
<td><div id="line106">To identify uncharacterized genes, we compared 5′ SAGE tags |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag107">bodyText</div></td>
<td><div id="line107">with RefSeq, EST and DBTSS databases. Of the 10,706 unique tags with |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag108">bodyText</div></td>
<td><div id="line108">a single locus in the genome, 9,376 corresponded to UniGene EST |XML| xmlLoc_0 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag109">bodyText</div></td>
<td><div id="line109">sequences and 7,800 corresponded to RefSeq sequences (Supplemen- |XML| xmlLoc_0 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag110">bodyText</div></td>
<td><div id="line110">tary Table 1). Furthermore, 6,418 unique 5′ SAGE tags corresponded to |XML| xmlLoc_0 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag111">bodyText</div></td>
<td><div id="line111">known genes in DBTSS. The remaining tags (12.4%) matched regions |XML| xmlLoc_0 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag112">bodyText</div></td>
<td><div id="line112">within introns (5.4%) of known genes or uncharacterized regions |XML| xmlLoc_0 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag113">bodyText</div></td>
<td><div id="line113">(6.6%). Tags that matched uncharacterized regions primarily hit two |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag114">bodyText</div></td>
<td><div id="line114">sites—completely uncharacterized regions or regions of uncharacter- |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag115">bodyText</div></td>
<td><div id="line115">ized EST sequences. Evidence of expression of such genes should help to |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag116">bodyText</div></td>
<td><div id="line116">uncover the genes’ full-length form by referring to 3′ SAGE. |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag117">bodyText</div></td>
<td><div id="line117">The SAGE method can be used to obtain quantitative transcript |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag118">bodyText</div></td>
<td><div id="line118">information. In the profile of the 5′-end transcripts in HEK293 cells |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag119">bodyText</div></td>
<td><div id="line119">(Supplementary Table 2), the most frequently expressed genes were |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag120">bodyText</div></td>
<td><div id="line120">identified as neurofilament 3 (NEF3 ), which had an expression frequency |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag121">bodyText</div></td>
<td><div id="line121">of 1.43%, followed by elongation factor 2. Several genes such as NEF3, |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag122">bodyText</div></td>
<td><div id="line122">heat shock 70 kDa protein 1A, calreticulin and heterogeneous nuclear |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag123">bodyText</div></td>
<td><div id="line123">ribonucleoprotein H1 were found using different tags. It is theorized |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag124">bodyText</div></td>
<td><div id="line124">that certain genes can be transcribed from different TSS. For example, |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag125">bodyText</div></td>
<td><div id="line125">our data suggest that both calreticulin and lactase dehydrogenase A can |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag126">bodyText</div></td>
<td><div id="line126">be transcribed from seven different transcriptional start sites (Fig. 2b). |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag127">bodyText</div></td>
<td><div id="line127">We performed Long SAGE on the 3′ ends of mRNA in HEK293 cells |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag128">bodyText</div></td>
<td><div id="line128">to validate the accuracy of our 5′ SAGE results. Using 3′ Long SAGE, |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag129">bodyText</div></td>
<td><div id="line129">we characterized 81,212 transcript tags. A total of 54,050 tags match- |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag130">bodyText</div></td>
<td><div id="line130">ed genomic sequences representing 15,423 different tags (Table 1). |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag131">bodyText</div></td>
<td><div id="line131">Seventy-five percent (11,613 tags) of 15,423 different tags matched one |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag132">bodyText</div></td>
<td><div id="line132">site in the genome. Furthermore, 8,359 3′ SAGE tags were found to |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag133">bodyText</div></td>
<td><div id="line133">match known genes in the UniGene EST database, and 5,267 3′ SAGE |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag134">bodyText</div></td>
<td><div id="line134">tags matched RefSeq genes (Supplementary Table 1). Nine percent of |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag135">bodyText</div></td>
<td><div id="line135">tags (1,395 tags) matched two loci in the genome, 13.2% (2,039 tags) |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag136">bodyText</div></td>
<td><div id="line136">matched 3–99 loci and 2.4% (376 tags) matched &gt;1 00 loci. The percent- |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag137">bodyText</div></td>
<td><div id="line137">age of tags that matched multiple sites in the genome was very similar |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag138">bodyText</div></td>
<td><div id="line138">for 5′ SAGE and 3′ SAGE (Table 1). On other hand, 5′ SAGE tags were |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag139">bodyText</div></td>
<td><div id="line139">very heterogeneous whereas 3′ SAGE tags were not. |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag140">bodyText</div></td>
<td><div id="line140">It has been shown that tags present at &gt;10 copies per genome are on |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag141">bodyText</div></td>
<td><div id="line141">average more highly expressed than those present at only one copy per |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag142">bodyText</div></td>
<td><div id="line142">genome 2. Similarly, our data demonstrated that the relative expression |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag143">bodyText</div></td>
<td><div id="line143">level was higher in 3 to ∼99 loci/genome than in other fractions in the |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag144">bodyText</div></td>
<td><div id="line144">5′ SAGE and 3′ SAGE libraries, because of association between gene |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag145">bodyText</div></td>
<td><div id="line145">expression and gene duplication through retrotransposition. |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag146">bodyText</div></td>
<td><div id="line146">To estimate the extent of the similarity between the two libraries, we |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag147">bodyText</div></td>
<td><div id="line147">compared the expressed genes of 5′ SAGE and 3′ Long SAGE. Because |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag148">bodyText</div></td>
<td><div id="line148">5′ and 3′ tags were randomly sampled separately from their 5′ and 3′ ends, |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag149">bodyText</div></td>
<td><div id="line149">the probability that 5′ tags could have associated with a particular full- |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag150">bodyText</div></td>
<td><div id="line150">length cDNA sequence was expected to coincide with the probability that |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag151">bodyText</div></td>
<td><div id="line151">3′ tags matched the cDNA. However, owing to incomplete collection of |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag152">bodyText</div></td>
<td><div id="line152">full-length cDNA sequences or alternatively spliced transcripts, it was |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag153">bodyText</div></td>
<td><div id="line153">not easy to determine the exact correspondence between 5′ and 3′ tags |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag154">bodyText</div></td>
<td><div id="line154"><span style="background-color: yellow;">even though they might have originated from the same coding region. |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</span></div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag155">tableCaption</div></td>
<td><div id="line155"><span style="background-color: yellow;">Table 1 Experimental matching of SAGE tag to genome |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_no xmlTable_yes xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</span></div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag156">table</div></td>
<td><div id="line156">5′-end SAGE tags to genome1	3′-end SAGE tags to genome2 |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_no xmlTable_yes xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag157">table</div></td>
<td><div id="line157">Tag loci in genome3	Tags mapped to	Unique tags mapped	Relative	Tags mapped to	Unique tags mapped	Relative |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_no xmlTable_yes xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag158">table</div></td>
<td><div id="line158">genome (%)	to genome (%)	expression level	genome (%)	to genome (%)	expression level |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_no xmlTable_yes xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag159">table</div></td>
<td><div id="line159">1 loci/genome	15,448 (77.7)	10,706 (79.9)	1.44	34,139 (63.2)	11,613 (75.3)	2.94 |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_no xmlTable_yes xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag160">table</div></td>
<td><div id="line160">2 loci/genome	2,037 (10.2)	1,483 (11.1)	1.37	6,739 (12.5)	1,395 (9.0)	4.83 |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_no xmlTable_yes xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag161">table</div></td>
<td><div id="line161">3–99 loci/genome	2,275 (11.4)	1,090 (8.1)	2.09	12,265 (22.7)	2,039 (13.2)	6.02 |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_no xmlTable_yes xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag162">table</div></td>
<td><div id="line162">&gt;100 loci/genome	133 (0.7)	125 (0.9)	1.06	907 (1.7)	376 (2.4)	2.42 |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_no xmlTable_yes xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag163">table</div></td>
<td><div id="line163"><span style="background-color: yellow;">Total tags	19,893 (100)	13,404 (100)	1.40	54,050 (100)	15,422 (100)	2.13 |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_no xmlTable_yes xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</span></div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag164">tableCaption</div></td>
<td><div id="line164">1Number of 18-bp 5′ SAGE tags getting hits in genome; 5,791 tags of 25,684 tags sequenced did not get hits. Relative expression level was determined by dividing the total number |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag165">tableCaption</div></td>
<td><div id="line165">of transcript tags observed in the library by the number of different tags. 2Number of 20-bp 3′ SAGE tags getting hits in genome; 27,162 tags of 81,211 sequenced did not get hits. |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag166">tableCaption</div></td>
<td><div id="line166"><span style="background-color: yellow;">3Number of genome locations matched by individual SAGE tags. |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</span></div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag167">footnote</div></td>
<td><div id="line167"><span style="background-color: yellow;">NATURE BIOTECHNOLOGY VOLUME 22 NUMBER 9 SEPTEMBER 2004	1147 |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_yes xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</span></div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag168">note</div></td>
<td><div id="line168"><span style="background-color: yellow;">LETTERS |XML| xmlLoc_0 xmlBold_no xmlItalic_no xmlFontSize_largest-1 xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</span></div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag169">table</div></td>
<td><div id="line169">Table 2 Distance of 5′ SAGE tags relative to mRNA start sites in |XML| xmlLoc_0 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_yes xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag170">table</div></td>
<td><div id="line170"><span style="background-color: yellow;">each database |XML| xmlLoc_0 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_yes xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</span></div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag171">figure</div></td>
<td><div id="line171"><span style="background-color: yellow;">a |XML| xmlLoc_0 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_yes xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</span></div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag172">figureCaption</div></td>
<td><div id="line172">Figure 2 5′ SAGE tags hit around the defined transcription start sites. |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_yes xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag173">figureCaption</div></td>
<td><div id="line173">(a) Distance of 5′ SAGE tags relative to the mRNA start sites in the UniGene |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_yes xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag174">figureCaption</div></td>
<td><div id="line174">and DBTSS sequences. Distances are shown as the number of upstream (–) |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_yes xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag175">figureCaption</div></td>
<td><div id="line175">and downstream (+) nucleotides (x-axis). The mRNA start site is depicted as |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_yes xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag176">figureCaption</div></td>
<td><div id="line176">0. The frequency of the 5′ SAGE tag is given on the y-axis. A small distance |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_yes xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag177">figureCaption</div></td>
<td><div id="line177">between the aligned positions of each 5′ SAGE tag and their corresponding |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_yes xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag178">figureCaption</div></td>
<td><div id="line178">gene implies that the 5′ tags are almost consistent with the known 5′ |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_yes xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag179">figureCaption</div></td>
<td><div id="line179">transcriptional start site. We used UniGene and DBTSS databases separately |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_yes xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag180">figureCaption</div></td>
<td><div id="line180"><span style="background-color: yellow;">to determine their coverage differences. (b) mRNA start sites and frequency. |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_yes xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</span></div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag181">figure</div></td>
<td><div id="line181"><span style="background-color: yellow;">b |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_yes xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</span></div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag182">bodyText</div></td>
<td><div id="line182">One possible approach would be to put together EST alignments that |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag183">bodyText</div></td>
<td><div id="line183">shared exons in common, treat such a cluster as a gene coding locus, |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag184">bodyText</div></td>
<td><div id="line184">map 5′ and 3′ SAGE tags to these clusters and their upstream regions and |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag185">bodyText</div></td>
<td><div id="line185">then uncover correspondences between 5′ and 3′ SAGE tag expression. |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag186">bodyText</div></td>
<td><div id="line186">Using this approach, we counted pairs of 3′ and 5′ tags for each gene |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag187">bodyText</div></td>
<td><div id="line187">coding region (Supplementary Fig. 2). Comparison of expression pat- |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag188">bodyText</div></td>
<td><div id="line188">terns revealed that most genes were expressed at similar levels in both |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag189">bodyText</div></td>
<td><div id="line189">libraries, though several transcripts were expressed at substantially dif- |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag190">bodyText</div></td>
<td><div id="line190">ferent levels. Pearson correlation coefficients of 5′ SAGE and 3′ SAGE |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag191">bodyText</div></td>
<td><div id="line191">libraries showed moderate similarity (0.36). This moderate correlation |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag192">bodyText</div></td>
<td><div id="line192">was probably due to dispersion of the frequencies from the 5′ SAGE and |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag193">bodyText</div></td>
<td><div id="line193">3′ SAGE libraries. There are several possible reasons for the appearance |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag194">bodyText</div></td>
<td><div id="line194">of these tags, including: (i) PCR amplification errors in 5′ SAGE and |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag195">bodyText</div></td>
<td><div id="line195">3′ SAGE; (ii) appropriation of the NlaIII restriction site in 3′ SAGE by a |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag196">bodyText</div></td>
<td><div id="line196">small number of genes; (iii) appropriation of the XhoI restriction site in |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag197">bodyText</div></td>
<td><div id="line197">5′ SAGE by a small number of genes; (iv) the presence of unknown |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag198">bodyText</div></td>
<td><div id="line198">splicing variants of mRNA in 5′ SAGE and 3′ SAGE; and (v) an annota- |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag199">bodyText</div></td>
<td><div id="line199">tion error for tags hitting multiple genomic loci or an EST-annotation |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag200">bodyText</div></td>
<td><div id="line200">error in the genome. |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag201">bodyText</div></td>
<td><div id="line201">This study identified only a fraction of the genes expressed in HEK293 |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag202">bodyText</div></td>
<td><div id="line202"><span style="background-color: yellow;">cells. A much larger number of tags from a variety of different cell types |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</span></div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag203">table</div></td>
<td><div id="line203">Approximate distance 	from the	Tag number (%) |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_no xmlTable_yes xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag204">table</div></td>
<td><div id="line204">start site of each database |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_no xmlTable_yes xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag205">table</div></td>
<td><div id="line205">(nt)	RefSeq	UniGene (GRL)	DBTSS |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_no xmlTable_yes xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag206">table</div></td>
<td><div id="line206">–500 to –201	349	(3.2)	204	(1.5)	160	(1.6) |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_no xmlTable_yes xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag207">table</div></td>
<td><div id="line207">–200 to –51	887	(8.1)	335	(2.4)	253	(2.5) |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_no xmlTable_yes xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag208">table</div></td>
<td><div id="line208">–50 to –1	4,179	(38.1)	3,957	(28.8)	1,965	(19.5) |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_no xmlTable_yes xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag209">table</div></td>
<td><div id="line209">0 to +50	3,173	(28.9)	8,673	(63.2)	7,149	(70.8) |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_no xmlTable_yes xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag210">table</div></td>
<td><div id="line210">+51 to +200	837	(7.6)	311	(2.3)	209	(2.1) |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_no xmlTable_yes xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag211">table</div></td>
<td><div id="line211">(–500 to +200)	9,425	(85.8)	13,480	(98.2)	9,736	(96.4) |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_no xmlTable_yes xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag212">table</div></td>
<td><div id="line212"><span style="background-color: yellow;">Total tags	10,982	(100)	13,723	(100)	10,098	(100) |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_no xmlTable_yes xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</span></div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag213">tableCaption</div></td>
<td><div id="line213">The tags that correspond in mapping with the 5′ ends of genes from each database |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag214">tableCaption</div></td>
<td><div id="line214"><span style="background-color: yellow;">were analyzed as described in Figure 2. |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</span></div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag215">bodyText</div></td>
<td><div id="line215">cultured under varying environmental conditions will be required to |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag216">bodyText</div></td>
<td><div id="line216">more thoroughly describe the compendium of expressed genes. |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag217">bodyText</div></td>
<td><div id="line217">Several groups reported that mRNA start sites9 and polyadenylation |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag218">bodyText</div></td>
<td><div id="line218">cleavage sites10 show heterogeneity. Although TSS differences have been |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag219">bodyText</div></td>
<td><div id="line219">reported for specific genes in tissues1 1, our data showed that the diver- |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag220">bodyText</div></td>
<td><div id="line220">sity of TSS already exists on a cellular level. Moreover, our data pro- |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag221">bodyText</div></td>
<td><div id="line221">vide direct evidence of heterogeneity of TSS and 3′ end regions using |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag222">bodyText</div></td>
<td><div id="line222">5′ SAGE and 3′ SAGE methods. For example, we found that the peroxi- |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag223">bodyText</div></td>
<td><div id="line223">some proliferator-activated receptor–binding protein has one TSS and |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag224">bodyText</div></td>
<td><div id="line224">two 3′ SAGE tag sites, ribosomal protein S4 has 16 TSS and one 3′ SAGE |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag225">bodyText</div></td>
<td><div id="line225">tag site, and calreticulin has seven TSS and one 3′ SAGE tag site. |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag226">bodyText</div></td>
<td><div id="line226">Alternative mRNA splicing contributes in important ways to the |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag227">bodyText</div></td>
<td><div id="line227">complexity of the human proteome. Recent genomic studies suggest |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag228">bodyText</div></td>
<td><div id="line228">that 40%–60% of human genes are alternatively spliced 12. Fifteen |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag229">bodyText</div></td>
<td><div id="line229">percent of point mutations are estimated to cause human genetic dis- |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag230">bodyText</div></td>
<td><div id="line230">eases by triggering an mRNA splicing defect13. It has been reported that |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag231">bodyText</div></td>
<td><div id="line231">49% of the transcriptional units with multiple splice forms included |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag232">bodyText</div></td>
<td><div id="line232">transcripts in which usage of an alternative TSS was accompanied by |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag233">bodyText</div></td>
<td><div id="line233">alternative splicing of the initial exon14. We also found that the mRNA |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag234">bodyText</div></td>
<td><div id="line234">start sites of several genes, such as peroxiredoxin 4(NM_006406), repre- |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag235">bodyText</div></td>
<td><div id="line235">sented not only different splicing variants of mRNA but also different |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag236">bodyText</div></td>
<td><div id="line236">degrees of gene expression, suggesting that alternative transcription may |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag237">bodyText</div></td>
<td><div id="line237">frequently induce alternative splicing. |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag238">bodyText</div></td>
<td><div id="line238">In conclusion, our 5′ SAGE method should facilitate the annotation |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag239">bodyText</div></td>
<td><div id="line239">of genomes. Because this method represents one of the few high- |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag240">bodyText</div></td>
<td><div id="line240">throughput discovery approaches that does not depend on a priori |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag241">bodyText</div></td>
<td><div id="line241">knowledge of gene sequences, it can be used for independent valida- |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag242">bodyText</div></td>
<td><div id="line242">tion of in silico gene predictions and for the identification of nonanno- |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag243">bodyText</div></td>
<td><div id="line243">tated regions. In addition, it should be useful for finding single |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag244">bodyText</div></td>
<td><div id="line244">nucleotide polymorphisms in 5′-untranslated or promoter regions. |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag245">bodyText</div></td>
<td><div id="line245">Considering the diversity of 5′ ends, it is more appropriate to perform |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag246">bodyText</div></td>
<td><div id="line246">5′ SAGE rather than 3′ SAGE when determining the frequency of gene |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag247">bodyText</div></td>
<td><div id="line247">expression. Comprehensive identification of genes transcribed from |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag248">bodyText</div></td>
<td><div id="line248">specific mRNA start sites in different cell types will not only provide |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag249">bodyText</div></td>
<td><div id="line249">insight into the functional complexity of the human genome but may |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag250">bodyText</div></td>
<td><div id="line250">also aid the diagnosis of various disorders such as cancer and immuno- |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag251">bodyText</div></td>
<td><div id="line251"><span style="background-color: yellow;">logical and neural diseases. |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</span></div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag252">sectionHeader</div></td>
<td><div id="line252"><span style="background-color: yellow;">METHODS |XML| xmlLoc_6 xmlBold_yes xmlItalic_no xmlFontSize_common xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</span></div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag253">bodyText</div></td>
<td><div id="line253">Generation of 3′-Long SAGE library. Total RNA was prepared from HEK239 cells |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag254">bodyText</div></td>
<td><div id="line254">and mRNA was isolated as previously described15. Long SAGE was performed as |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag255">bodyText</div></td>
<td><div id="line255">previously described2. SAGE 2000 software (version 4.12) was used to quantify |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag256">bodyText</div></td>
<td><div id="line256">the abundance of each tag. After elimination of the linker sequences, other poten- |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag257">bodyText</div></td>
<td><div id="line257">tial artifacts and repeated ditags, each tag was analyzed. |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag258">bodyText</div></td>
<td><div id="line258">Generation of 5′ SAGE library. Oligo-capping was done as described by |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag259">bodyText</div></td>
<td><div id="line259"><span style="background-color: yellow;">Maruyama and Sugano7 with some modifications16. Briefly, 5–10 g of poly(A)+ |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</span></div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag260">note</div></td>
<td><div id="line260"><span style="background-color: yellow;">1148	VOLUME 22 NUMBER 9 SEPTEMBER 2004 NATURE BIOTECHNOLOGY |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_yes xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</span></div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag261">note</div></td>
<td><div id="line261"><span style="background-color: yellow;">LETTERS |XML| xmlLoc_0 xmlBold_no xmlItalic_no xmlFontSize_largest-1 xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</span></div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag262">bodyText</div></td>
<td><div id="line262">RNA were treated with 1.2 units of bacterial alkaline phosphatase (BAP; TaKaRa) |XML| xmlLoc_0 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag263">bodyText</div></td>
<td><div id="line263">in 100 l of 100 mM Tris-HCl (pH 8.0) containing 5 mM 2-mercaptoethanol and |XML| xmlLoc_0 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag264">bodyText</div></td>
<td><div id="line264">100 units of RNAsin (Promega) at 37 �C for 40 min. After extraction twice with |XML| xmlLoc_0 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag265">bodyText</div></td>
<td><div id="line265">phenol:chloroform (1:1) and ethanol precipitation, the poly(A)+ RNA was |XML| xmlLoc_0 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag266">bodyText</div></td>
<td><div id="line266">treated with 20 units of tobacco acid pyrophosphatase (TAP) in 100 l of 50 mM |XML| xmlLoc_0 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag267">bodyText</div></td>
<td><div id="line267">sodium acetate (pH 5.5), 1 mM EDTA, 5 mM 2-mercaptoethanol and 100 units |XML| xmlLoc_0 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag268">bodyText</div></td>
<td><div id="line268">of RNAsin at 37 �C for 45 min. After phenol:chloroform extraction and ethanol |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag269">bodyText</div></td>
<td><div id="line269">precipitation, 2–4 g of the BAP-TAP-treated poly(A)+ RNA were divided into |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag270">bodyText</div></td>
<td><div id="line270">two pools, and one of the following RNA linkers containing recognition sites for |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag271">bodyText</div></td>
<td><div id="line271">XhoI/MmeI was ligated to each pool—5′ -oligo 1 (5′-UUU GGA UUU GCU GGU |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag272">bodyText</div></td>
<td><div id="line272">GCA GUA CAA CUA GGC UUA AUA CUC GAG UCC GACG -3′) or 5′ -oligo 2 |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag273">bodyText</div></td>
<td><div id="line273">(5′-UUU CTG CUC GAA UUC AAG CUU CUA ACG AUG UAC GCU CGA |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag274">bodyText</div></td>
<td><div id="line274">GUC CGA CG -3′)—using 250 units of RNA ligase (TaKaRa) in 100 ml of 50 mM |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag275">bodyText</div></td>
<td><div id="line275">Tris-HCl (pH 7.5), 5 mM MgCl2, 5 mM 2-mercaptoethanol, 0.5 mM ATP, |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag276">bodyText</div></td>
<td><div id="line276">25% PEG8000 and 100 units of RNAsin at 20 °C for 3–16 h. |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag277">bodyText</div></td>
<td><div id="line277">After removing unligated 5′-oligo, cDNA was synthesized with RNaseH-free |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag278">bodyText</div></td>
<td><div id="line278">reverse-transcriptase (Superscript II, Gibco BRL). For the 5′-end-enriched cDNA |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag279">bodyText</div></td>
<td><div id="line279">library, 10 pmol of random adapter-primer (5′-GCG GCT GAA GAC GGC CTA |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag280">bodyText</div></td>
<td><div id="line280">TGT GGC CNN N NC-3′) was used and incubation was carried out at 12 °C for |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag281">bodyText</div></td>
<td><div id="line281">1 h and 42 �C for another hour. |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag282">bodyText</div></td>
<td><div id="line282">After first-strand synthesis, RNA was degraded in 15 mM NaOH at 65 �C for |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag283">bodyText</div></td>
<td><div id="line283">1 h. The cDNA that was made from 10 g of oligo-capped poly(A)+ RNA was |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag284">bodyText</div></td>
<td><div id="line284">amplified in a volume of 100 l using an XL PCR kit (Perkin-Elmer) with |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag285">bodyText</div></td>
<td><div id="line285">16 pmol of 5′ (5′ biotin- GGA TTT GCT GGT GCA GTA CAA CTA GGC TTA |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag286">bodyText</div></td>
<td><div id="line286">ATA-3′ or 5′ biotin- CTG CTC GAA TTC AAG CTT CTA ACG ATG TAC G-3′) |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag287">bodyText</div></td>
<td><div id="line287">and 3′ (5′-GCG GCT GAA GAC GGC CTA TGT-3′) PCR primers. Random |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag288">bodyText</div></td>
<td><div id="line288">adapter-primer-primed cDNA was amplified for 10 cycles at 94 �C for 1 min, |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag289">bodyText</div></td>
<td><div id="line289">58 �C for 1 min and 72 �C for 2 min. PCR products were extracted with |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag290">bodyText</div></td>
<td><div id="line290">phenol:chloroform (1:1) once, ethanol-precipitated and digested with the MmeI |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag291">bodyText</div></td>
<td><div id="line291">type-IIS restriction endonuclease (University of Gdansk Center for Technology |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag292">bodyText</div></td>
<td><div id="line292">Transfer). Digestion was performed at 37 �C for 2.5 h using 40 units MmeI in |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag293">bodyText</div></td>
<td><div id="line293">300 l of 10 mM HEPES, pH 8.0, 2.5 mM potassium acetate, 5 mM magnesium |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag294">bodyText</div></td>
<td><div id="line294">acetate, 2 mM DTT and 40 M S-adenosylmethionine. The digested 5′-terminal |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag295">bodyText</div></td>
<td><div id="line295">cDNA fragments were bound to streptavidin-coated magnetic beads (Dynal). |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag296">bodyText</div></td>
<td><div id="line296">cDNA fragments that bound to the beads were directly ligated together in 16 l of |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag297">bodyText</div></td>
<td><div id="line297">reaction mixture containing four units of T4 DNA ligase in the supplied buffer |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag298">bodyText</div></td>
<td><div id="line298">for 2.5 h at 16 °C. The ditags were amplified by PCR using primers 5′ -GGA TTT |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag299">bodyText</div></td>
<td><div id="line299">GCT GGT GCA GTA CAA CTA GGC-3′ and 5′-CTG CTC GAA TTC AAG CTT |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag300">bodyText</div></td>
<td><div id="line300">CTA ACG ATG-3′. The PCR products were analyzed by PAGE and digested with |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag301">bodyText</div></td>
<td><div id="line301">XhoI. The band containing the ditags was excised and self-ligated to produce long |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag302">bodyText</div></td>
<td><div id="line302">concatemers which were then cloned into the XhoI site of pZero 1.0 (Invitrogen). |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag303">bodyText</div></td>
<td><div id="line303">Colonies were screened with PCR using M13 forward and reverse primers. PCR |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag304">bodyText</div></td>
<td><div id="line304">products containing inserts of more than 600 bp were sequenced with the Big |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag305">bodyText</div></td>
<td><div id="line305">Dye terminator ver.3 and analyzed using a 3730 ABI automated DNA sequencer |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag306">bodyText</div></td>
<td><div id="line306">(Applied Biosystems). All electropherograms were reanalyzed by visual inspec- |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag307">bodyText</div></td>
<td><div id="line307">tion to check for ambiguous bases and to correct misreads. SAGE 2000 software |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag308">bodyText</div></td>
<td><div id="line308">(version 4.12) was used to quantify the abundance of each tag. In this study, we |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag309">bodyText</div></td>
<td><div id="line309">obtained 18–19 bp tag information. 5′ SAGE and 3′ SAGE data are available at |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag310">bodyText</div></td>
<td><div id="line310">http://sage.gi.k.u-tokyo.ac.jp/. |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag311">bodyText</div></td>
<td><div id="line311">Generation of captured full-length cDNA library. The captured full-length |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag312">bodyText</div></td>
<td><div id="line312">cDNA library in HEL293 was constructed using the oligo-capping method as |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag313">bodyText</div></td>
<td><div id="line313">described before16. The 5′ ends of 7,903 clones sequenced were analyzed as well as |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag314">bodyText</div></td>
<td><div id="line314">5′ SAGE tags. The raw sequencing data are deposited in DDBJ and are also avail- |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag315">bodyText</div></td>
<td><div id="line315">able as a Supplementary Note online and at http://sage.gi.k.u-tokyo.ac.jp/. |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag316">bodyText</div></td>
<td><div id="line316">Association of 5′ SAGE tag with corresponding genes. To assess the validity of |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag317">bodyText</div></td>
<td><div id="line317">5′ SAGE tags for identifying transcriptional start sites, we avoided aligning |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag318">bodyText</div></td>
<td><div id="line318">5′ SAGE tags (18 bp) with the current cDNA/EST database because the sequences |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag319">bodyText</div></td>
<td><div id="line319">were not always read from their transcriptional start sites. We instead attempted |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag320">bodyText</div></td>
<td><div id="line320">to align our 5′-tags with the human genome sequence, the NCBI build 34 avail- |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag321">bodyText</div></td>
<td><div id="line321">able from http://genome.ucsc.edu/, by using the alignment program ALPS that is |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag322">bodyText</div></td>
<td><div id="line322">publicly available from http://alps.gi.k.u-tokyo.ac.jp/. Only tags that matched in |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag323">bodyText</div></td>
<td><div id="line323">the sense orientation were considered for this analysis. |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag324">bodyText</div></td>
<td><div id="line324">The regions adjacent to the alignment location of each 5′-tag were searched to |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag325">bodyText</div></td>
<td><div id="line325">find their corresponding transcript by using the Gene Resource Locator data- |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag326">bodyText</div></td>
<td><div id="line326">base17, a database of sequence alignments from various resources such as |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag327">bodyText</div></td>
<td><div id="line327">UniGene (Build 162)18. A major problem that we ran into was that, owing to |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag328">bodyText</div></td>
<td><div id="line328">retro-transposition and genome-duplication, one 5′-tag could be aligned with |XML| xmlLoc_0 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag329">bodyText</div></td>
<td><div id="line329">multiple locations, though many of these were noncoding regions. The issue was |XML| xmlLoc_0 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag330">bodyText</div></td>
<td><div id="line330">resolved by selecting gene-coding locations that were annotated in the UniGene |XML| xmlLoc_0 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag331">bodyText</div></td>
<td><div id="line331">database. Although 3′ -tags often fell in the 3′-end exons, 5′-tags did not necessar- |XML| xmlLoc_0 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag332">bodyText</div></td>
<td><div id="line332">ily hit the first exons. Thus the search was made within 500 bp from the alignment |XML| xmlLoc_0 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag333">bodyText</div></td>
<td><div id="line333">position of each 5′-tag. |XML| xmlLoc_0 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag334">bodyText</div></td>
<td><div id="line334">Consistency with known 5′ transcriptional start sites. A small distance between |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag335">bodyText</div></td>
<td><div id="line335">the aligned positions of each 5′ SAGE tag and its corresponding gene implied that |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag336">bodyText</div></td>
<td><div id="line336">the 5′ tag was nearly consistent with the known 5′ transcriptional start site.To cal- |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag337">bodyText</div></td>
<td><div id="line337">culate the distance, however, we had to bear in mind that, near the 5′ tag, multiple |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag338">bodyText</div></td>
<td><div id="line338">cDNA/EST sequence alignments might be observed because of alternative splic- |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag339">bodyText</div></td>
<td><div id="line339">ing. To resolve this problem and assign a unique value to the distance, we selected |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag340">bodyText</div></td>
<td><div id="line340">the alignment that was closest to the 5′ tag. The distance was defined as being neg- |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag341">bodyText</div></td>
<td><div id="line341">ative if the 5′ tag was located in the upstream region of its corresponding cDNA. |XML| xmlLoc_1 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag342">bodyText</div></td>
<td><div id="line342">Otherwise, the value was deemed positive or zero; the zero distance indicated the |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag343">bodyText</div></td>
<td><div id="line343">perfect coincidence. To observe the overall distance distribution, we calculated |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag344">bodyText</div></td>
<td><div id="line344">the total number of 5′ SAGE tag occurrences between –500 to +200 nt of the |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag345">bodyText</div></td>
<td><div id="line345">mRNA start sites. We used RefSeq, UniGene(GRL) and DBTSS databases sepa- |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag346">bodyText</div></td>
<td><div id="line346"><span style="background-color: yellow;">rately to determine differences in their coverage of the transcriptional start sites. |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</span></div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag347">note</div></td>
<td><div id="line347"><span style="background-color: yellow;">Note: Supplementary information is available on the Nature Biotechnology website. |XML| xmlLoc_2 xmlBold_no xmlItalic_yes xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</span></div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag348">sectionHeader</div></td>
<td><div id="line348"><span style="background-color: yellow;">ACKNOWLEDGMENTS |XML| xmlLoc_2 xmlBold_yes xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</span></div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag349">bodyText</div></td>
<td><div id="line349">This work was supported by Grant-in-Aid for Scientific Research on Priority Areas |XML| xmlLoc_2 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag350">bodyText</div></td>
<td><div id="line350">(C) “Medical Genome Science” from the Ministry of Education, Culture, Sports, |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag351">bodyText</div></td>
<td><div id="line351"><span style="background-color: yellow;">Science and Technology of Japan. |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_continue
</span></div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag352">sectionHeader</div></td>
<td><div id="line352"><span style="background-color: yellow;">COMPETING INTERESTS STATEMENT |XML| xmlLoc_3 xmlBold_yes xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</span></div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag353">bodyText</div></td>
<td><div id="line353"><span style="background-color: yellow;">The authors declare that they have no competing financial interests. |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</span></div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag354">note</div></td>
<td><div id="line354">Received 20 January; accepted 7 June 2004 |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag355">note</div></td>
<td><div id="line355"><span style="background-color: yellow;">Published online at http://www.nature.com/naturebiotechnology/ |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_continue bi_xmlPara_new
</span></div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag356">reference</div></td>
<td><div id="line356">1. Duggan, D.J., Bittner, M., Chen, Y., Meltzer, P. &amp; Trent, J.M. Expression profiling |XML| xmlLoc_3 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag357">reference</div></td>
<td><div id="line357">using cDNA microarrays. Nat. Genet. 21, 10–14 (1999). |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag358">reference</div></td>
<td><div id="line358">2. Saha, S. et al. Using the transcriptome to annotate the genome. Nat. Biotechnol. 20, |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag359">reference</div></td>
<td><div id="line359">508–512 (2002). |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag360">reference</div></td>
<td><div id="line360">3. Madden, S.L., Galella, E.A., Zhu, J., Bertelsen, A.H. &amp; Beaudry, G.A. SAGE transcript |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag361">reference</div></td>
<td><div id="line361">profiles for p53-dependent growth regulation. Oncogene 15, 1079–1085 (1997). |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag362">reference</div></td>
<td><div id="line362">4. Velculescu, V.E. et al. Analysis of human transcriptomes. Nat. Genet. 23, 387–388 |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag363">reference</div></td>
<td><div id="line363">(1999). |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag364">reference</div></td>
<td><div id="line364">5. Hashimoto, S. et al. Gene expression profile in human leukocytes. Blood 101, |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag365">reference</div></td>
<td><div id="line365">3509–3513 (2003). |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag366">reference</div></td>
<td><div id="line366">6. Lander, E.S. et al. Initial sequencing and analysis of the human genome. Nature409, |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag367">reference</div></td>
<td><div id="line367">860–921 (2001). |XML| xmlLoc_4 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag368">reference</div></td>
<td><div id="line368">7. Maruyama, K. &amp; Sugano, S. Oligo-capping: a simple method to replace the cap struc- |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag369">reference</div></td>
<td><div id="line369">ture of eukaryotic mRNAs with oligoribonucleotides. Gene 138, 171–174 (1994). |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag370">reference</div></td>
<td><div id="line370">8. Suzuki, Y. et al. DBTSS: DataBase of human Transcriptional Start Sites and full- |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag371">reference</div></td>
<td><div id="line371">length cDNAs. Nucleic Acids Res. 30, 328–331 (2002). |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag372">reference</div></td>
<td><div id="line372">9. Suzuki, Y. et al. Diverse transcriptional initiation revealed by fine, large-scale map- |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag373">reference</div></td>
<td><div id="line373">ping of mRNA start sites. EMBO Rep. 2, 388–393 (2001). |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag374">reference</div></td>
<td><div id="line374">10. Pauws, E., van Kampen, A.H., van de Graaf, S.A., de Vijlder, J.J. &amp; Ris-Stalpers, C. |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag375">reference</div></td>
<td><div id="line375">Heterogeneity in polyadenylation cleavage sites in mammalian mRNA sequences: |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag376">reference</div></td>
<td><div id="line376">implications for SAGE analysis. Nucleic Acids Res. 29, 1690–1694 (2001). |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag377">reference</div></td>
<td><div id="line377">11. Shiraki, T. et al. Cap analysis gene expression for high-throughput analysis of tran- |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag378">reference</div></td>
<td><div id="line378">scriptional starting point and identification of promoter usage. Proc. Natl. Acad. Sci. |XML| xmlLoc_5 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag379">reference</div></td>
<td><div id="line379">USA 100, 15776–15781 (2003). |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag380">reference</div></td>
<td><div id="line380">12. Modrek, B. &amp; Lee, C. A genomic view of alternative splicing. Nat. Genet. 30, 13–19 |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag381">reference</div></td>
<td><div id="line381">(2002). |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag382">reference</div></td>
<td><div id="line382">13. Krawczak, M., Reiss, J. &amp; Cooper, D.N. The mutational spectrum of single base-pair |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag383">reference</div></td>
<td><div id="line383">substitutions in mRNA splice junctions of human genes: causes and consequences. |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag384">reference</div></td>
<td><div id="line384">Hum. Genet. 90, 41–54 (1992). |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag385">reference</div></td>
<td><div id="line385">14. Zavolan, M. et al. Impact of alternative initiation, splicing, and termination on the |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag386">reference</div></td>
<td><div id="line386">diversity of the mRNA transcripts encoded by the mouse transcriptome. Genome Res. |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag387">reference</div></td>
<td><div id="line387">13, 1290–1300 (2003). |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag388">reference</div></td>
<td><div id="line388">15. Hashimoto, S.-I., Suzuki, T., Dong, H.-Y., Yamazaki, N. &amp; Matsushima, K. Serial |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag389">reference</div></td>
<td><div id="line389">analysis of gene expression in human monocytes and macrophages. Blood 94, |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag390">reference</div></td>
<td><div id="line390">837–844 (1999). |XML| xmlLoc_6 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag391">reference</div></td>
<td><div id="line391">16. Suzuki, Y., Yoshitomo-Nakagawa, K., Maruyama, K., Suyama, A. &amp; Sugano, S. Con- |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag392">reference</div></td>
<td><div id="line392">struction and characterization of a full length-enriched and a 5′-end-enriched cDNA |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag393">reference</div></td>
<td><div id="line393">library. Gene 200, 149–156 (1997). |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag394">reference</div></td>
<td><div id="line394">17. Honkura, T., Ogasawara, J., Yamada, T. &amp; Morishita, S. The Gene Resource Locator: |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_continue bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag395">reference</div></td>
<td><div id="line395">gene locus maps for transcriptome analysis. Nucleic Acids Res. 30, 221–225 |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag396">reference</div></td>
<td><div id="line396">(2002). |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_continue bi_xmlPara_continue
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag397">reference</div></td>
<td><div id="line397">18. Wheeler, D.L. Database Resources of the National Center for Biotechnology. Nucleic |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_new bi_xmlPara_new
</div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag398">reference</div></td>
<td><div id="line398"><span style="background-color: yellow;">Acids Res. 31, 28–33 (2003). |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_smaller xmlPic_no xmlTable_no xmlBullet_yes bi_xmlSFBIA_continue bi_xmlPara_continue
</span></div></td>
</tr>
<tr>
<td><div tagged="yes" id="tag399">footnote</div></td>
<td><div id="line399"><span style="background-color: yellow;">NATURE BIOTECHNOLOGY VOLUME 22 NUMBER 9 SEPTEMBER 2004	1149 |XML| xmlLoc_7 xmlBold_no xmlItalic_no xmlFontSize_none xmlPic_yes xmlTable_no xmlBullet_no bi_xmlSFBIA_new bi_xmlPara_new
</span></div></td>
</tr>
</tbody></table>
<div id="floatdiv" style="position: absolute; width: 300px; height: 40px; top: 15799px; padding: 16px; background: none repeat scroll 0% 0% rgb(255, 255, 255); border: 2px solid rgb(34, 102, 170); z-index: 100; left: 1079px;">
<center>
<select id="tagtype">
<option value="unknown">unknown</option>
<option value="address">address</option>
<option value="affiliation">affiliation</option>
<option value="author">author</option>
<option value="bodyText">bodyText</option>
<option value="category">category</option>
<option value="construct">construct</option>
<option value="copyright">copyright</option>
<option value="email">email</option>
<option value="equation">equation</option>
<option value="figure">figure</option>
<option value="figureCaption">figureCaption</option>
<option selected="selected" value="footnote">footnote</option>
<option value="keyword">keyword</option>
<option value="listItem">listItem</option>
<option value="note">note</option>
<option value="page">page</option>
<option value="reference">reference</option>
<option value="sectionHeader">sectionHeader</option>
<option value="subsectionHeader">subsectionHeader</option>
<option value="subsubsectionHeader">subsubsectionHeader</option>
<option value="table">table</option>
<option value="tableCaption">tableCaption</option>
<option value="title">title</option>
</select>
<input value="Tag" onclick="surroundRange()" type="button">
</center>
</div>
</body></html>